boards/arm/96b_argonkey/doc/96b_argonkey.rst

```diff
@@ -0,0 +1,164 @@
+.. _96b_argonkey:
+
+96Boards Argonkey
+################
```

Comment:
Underline on titles need to be at least as long as the title, so add another #

Reply:
ok